docs/scripts_js/EXAMPLE_CreateAssayFile.js

```diff
@@ -6,21 +6,6 @@
 
 // Import ARCtrl
 import * as arctrl from "@nfdi4plants/arctrl"
-// Alternative import
```

Comment:
Sure we should kick this out? This was actually meant as an alternative, not some leftover code

Reply:
this is very basic js syntax knowledge, not sure if we should keep and maintain it